src/components/offers/list/OffersView.js

```diff
@@ -8,16 +8,18 @@ import {Filter} from "../filter/Filter";
 import {offersAPI} from "../../../utils/apis/OfferApi";
 import {filterOffers} from "../../../utils/functions";
 import PropTypes from "prop-types";
+import {SorterWrapper} from "../sorter/SorterWrapper";
 
 export const OffersView = (props) => {
 
     const [selectedOffer, setSelectedOffer] = useState(null)
     const [offers, setOffers] = useState([])
     const {id} = useParams()
     const [fixedOffers, setFixedOffers] = useState([])
+    const [sort, setSort] = useState({apply: (offers) => offers})
```

Comment:
Jak to działa? Albo dokładniej gdzie przekazujesz tą funkcję?

Reply:
Funkcję przekazuję w funkcji `handleSort`. Tam ustawiam sorta i sortuję oferty - stąd ten wyścig tam. To chyba wyedytuję do tablicy po prostu, bo nie ma sensu tego trzymać jako obiektu.